src/app/+admin/admin-registries/bitstream-formats/bitstream-formats.component.ts

```diff
@@ -13,24 +19,122 @@ import { PaginationComponentOptions } from '../../../shared/pagination/paginatio
   selector: 'ds-bitstream-formats',
   templateUrl: './bitstream-formats.component.html'
 })
-export class BitstreamFormatsComponent {
+export class BitstreamFormatsComponent implements OnInit {
 
   /**
    * A paginated list of bitstream formats to be shown on the page
    */
   bitstreamFormats: Observable<RemoteData<PaginatedList<BitstreamFormat>>>;
 
+  /**
+   * A BehaviourSubject that keeps track of the pageState used to update the currently displayed bitstreamFormats
+   */
+  pageState: BehaviorSubject<string>;
+
+  /**
+   * The current pagination configuration for the page used by the FindAll method
+   * Currently simply renders all bitstream formats
+   */
+  config: FindAllOptions = Object.assign(new FindAllOptions(), {
+    elementsPerPage: 10000
```

Comment:
In the initial version of this feature, the page size was set to a huge number [because of rest side limitations](https://github.com/DSpace/dspace-angular/pull/246#issuecomment-391840390). Since the rest has been updated, please verify whether this is still needed.

Reply:
Fixed the pagination to show 20 items.